src/listener/Lists.php

```diff
@@ -104,11 +114,19 @@ public function render(Lexer $lexer)
                 $isOpen = false;
             }
 
+            // defines whether this attribute list element is a checked/unchecked list or not.
+            $isCheck = in_array($this->getPickType($pick), [self::LIST_TYPE_CHECKED, self::LIST_TYPE_UNCHECKED], true);
+
             // create the opening OL/UL tag
             // opening tag process has been simplified, see https://github.com/nadar/quill-delta-parser/pull/33
             // and https://github.com/nadar/quill-delta-parser/issues/30
             if (!$isOpen) {
-                $output .= '<'.$this->getListAttribute($pick).'>'.PHP_EOL;
+                $output .= '<'.$this->getListAttribute($pick);
+                if ($isCheck) {
+                    $output .= ' class="list-unstyled"';
```

Comment:
please add the class as property of the Lists.php class, so people can override the listener with its own class name. thanks!